src/Microsoft.Identity.Web.TokenCache/Distributed/MsalDistributedTokenCacheAdapterOptions.cs

```diff
@@ -53,5 +53,13 @@ public class MsalDistributedTokenCacheAdapterOptions : DistributedCacheEntryOpti
         /// </summary>
         /// The default is <c>false.</c>
         public bool DisableL1Cache { get; set; }
+
+        /// <summary>
+        /// Enable writing to the L2 cache to be async (fire and forget).
+        /// This improves performance as the MSAL.NET will not have to wait
+        /// for the write to complete.
+        /// </summary>
+        /// The default is <c>false.</c>
```

Comment:
Should we default to true here instead as per original proposal?

Reply:
@bgavrilMS @SameerK-MSFT:
I disagree, and I've asked @jennyf19 to set it to `false` for now. 
I want first to be proven that this won't break any of the 1000+ partners we migrated already. So I propose that:
1. We have things tested by a set of partners (including R9 / Office) who will set this to true explicitly.
2. When we see that things go will we'll change the default to `true` and that will benefit to everybody.

Reply:
Sounds reasonable.